src/app/teams/teams-view.component.ts

```diff
@@ -236,20 +236,20 @@ export class TeamsViewComponent implements OnInit, AfterViewChecked, OnDestroy {
   changeMembershipRequest(type, memberDoc?) {
     const changeObject = this.changeObject(type, memberDoc);
     return () => {
-      this.dialogsLoadingService.start();
       return changeObject.obs.pipe(
         switchMap(() => type === 'added' ? this.teamsService.removeFromRequests(this.team, memberDoc) : of({})),
         switchMap(() => this.getMembers()),
         switchMap(() => this.sendNotifications('added')),
         map(() => changeObject.message),
-        finalize(() => this.dialogsLoadingService.stop())
```

Comment:
Please keep this in the `finalize` operator.  This will ensure the dialog closes even if there is an error.